src/NServiceBus.SqlServer/NServiceBus.SqlServer.csproj

```
@@ -20,7 +20,7 @@
     <PackageReference Include="Obsolete.Fody" Version="5.2.1" PrivateAssets="All" />
     <PackageReference Include="Particular.CodeRules" Version="0.3.0" PrivateAssets="All" />
     <PackageReference Include="Particular.Packaging" Version="0.9.0" PrivateAssets="All" />
-    <PackageReference Include="NServiceBus" Version="[7.2.0, 8.0.0)" />
+    <PackageReference Include="NServiceBus" Version="[8.0.0-alpha.541,)" />
```

Was it intentional to have this unbound? And not
```suggestion
    <PackageReference Include="NServiceBus" Version="[8.0.0-alpha.541, 9.0.0)" />
```

---

Nope. Good catch!